.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+# Summary of Changes
+- Include a summary of changes made by this pull request
+
+## Pull Request Checklist
+
+### Is Content created by the story?
+- [ ] No
+- [ ] If yes, the content is:
+   - [ ] Accessible
+   - [ ] Mobile-friendly
+   - [ ] Version-controlled
+   - [ ] Governed by appropriate roles and permissions
+   - [ ] Organized by taxonomy
+
+### Are there User-facing changes?
+- [ ] No
+- [ ] If yes, then:
+   - [ ] User documentation is updated
+   - [ ] User training is updated
+
+### Are there Upstream Code changes?
+- [ ] No
+- [ ] If yes, then:
+   - [ ] Code is complete, commented, and mergeable against *develop* branch.
+   - [ ] Code is secure to the best of our knowledge and does not contain anything that cannot be on a public repo (e.g. passwords)
+   - Is a test necessary?
+      - [ ] No
+      - [ ] If yes, then:
+         - [ ] Add a test if existing tests do not effectively test the code change
+         - [ ] Test builds without error
+   - [ ] Pull request (PR) created to merge against *develop*
+   - [ ] PR has a meaningful title
+   - [ ] PR has a summary of changes
+   - [ ] PR is peer reviewed and meets team standards.
+   - [ ] PR builds without error
+   - [ ] PR merged with *develop*
```

Comment:
Is this a Catch-22? Can the branch be merged before all checks are passed?

Reply:
Yes - I've removed the "PR merged with develop" check.